client/src/components/Icons/icon-styles.css

```
@@ -0,0 +1,9 @@
+.faded {
+    filter: invert(73%) sepia(4%) saturate(555%) hue-rotate(171deg) brightness(95%) contrast(94%);
+}
+.primary {
+    filter: invert(26%) sepia(77%) saturate(6805%) hue-rotate(242deg) brightness(97%) contrast(96%);
+}
+.default {
+    filter: none;
+}
```

Just adding a few lines for readability.
```suggestion
.faded {
    filter: invert(73%) sepia(4%) saturate(555%) hue-rotate(171deg) brightness(95%) contrast(94%);
}

.primary {
    filter: invert(26%) sepia(77%) saturate(6805%) hue-rotate(242deg) brightness(97%) contrast(96%);
}

.default {
    filter: none;
}
```